docs/format/source/format_description.rst

```diff
@@ -203,6 +203,22 @@ Extending *NWBContainer* works in the same way, e.g., to create more specific ty
 data processing.
 
 
+Common attributes
+-----------------
+
+All NWB:N Groups and Datasets have three required attributes: `neurodata_type`, `namespace`, and `object_id`.
```

Comment:
Instead of "All NWB:N Groups and Datasets" I think we should say something like "All NWB:N Groups and Datasets with an assigned neurodata_type have ...."